doc/Dev.rst

```diff
@@ -172,11 +177,12 @@ A typical development workflow will be::
    cd ../sarracenia/test
    ./flow_cleanup.sh
    rm directories with state (indicated by flow_cleanup.sh)
-   ./flow_setup.sh  ; *starts the flows*
-   ./flow_limit.sh  ; *stops the flows after some period (default: 1000) *
-   ./flow_check.sh  ; *checks the flows*
-   ./flow_cleanup.sh  ; *cleans up the flows*
-   
+   ./flow_setup.sh  # *starts the flows*
+   ./flow_limit.sh  # *stops the flows after some period (default: 1000) *
+   ./flow_check.sh  # *checks the flows*
+   ./flow_cleanup.sh  # *cleans up the flows*
+   git commit -a  # on the branch...
```

Comment:
just a thought on the typical workflow, as one commit branch has never been the typical flow for me: 
- this is time consuming to run the flow test like that for each commit when there are a lot... and it doesn't add plus-value to the system quality. Defining our requirements for the issue and adding test case to our test suite (when needed) would improve much more the quality of our system. The flow test would be the final integration test before the pull request...